cocos/2d/components/label.ts

```diff
@@ -674,6 +674,16 @@ export class Label extends UIRenderer {
     protected _fontAtlas: FontAtlas | null = null;
     protected _letterTexture: LetterRenderTexture | null = null;
 
+    protected _contextWidth = 0;
+
+    get contextWidth() {
```

Comment:
What's contextWidth ? Add some comment for it ?

Reply:
Requires the length of the Label text content